src/physics/em/detail/MollerEnergyDistribution.hh

```diff
@@ -26,30 +26,32 @@ class MollerEnergyDistribution
   public:
     // Construct with data from MollerBhabhaInteractor
     inline CELER_FUNCTION
-    MollerEnergyDistribution(const MollerBhabhaPointers& shared,
-                             const real_type             inc_energy);
+    MollerEnergyDistribution(const real_type electron_mass_c_sq,
+                             const real_type min_valid_energy,
+                             const real_type inc_energy);
 
     // Sample the exiting energy
     template<class Engine>
     inline CELER_FUNCTION real_type operator()(Engine& rng);
 
   private:
-    // Electron mass * c^2 [MeV]
-    real_type electron_mass_c_sq_;
     // Electron incident energy [MeV]
     real_type inc_energy_;
-    // Total energy of the incident particle
+    // Total energy of the incident particle [MeV]
     real_type total_energy_;
     // Maximum energy fraction transferred to free electron
-    real_type max_energy_fraction_;
+    static CELER_CONSTEXPR_FUNCTION real_type max_energy_fraction()
+    {
+        return 0.5;
+    }
```

Comment:
Now that these are functions rather than member data, can you move them below the data members to visually separate them? Same with the Bhablah distribution.